explainer_worker/sql_db.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import select, update
+
+from tabels import Upload, _session, Status
+
+
```

Comment:
there is another file with the same name

Reply:
what??